docs/src/environments/_index.md

```diff
@@ -1,118 +1,165 @@
+
+### Workflows
+
+Since you can customize your [environment hierarchy](#hierarchy), you can create your own workflows. 
+You can decide which structure best fits your needs. 
+
+#### Possible approaches
+
+You may want to take one of the following approaches when creating your workflows:
+
+-   **Agile**: A child environment per sprint.
+    Each story in the sprint can have its own environment as a child of the sprint environment.
+
+-   **Developer-centric**: One QA environment and a few development environments
+    (*per developer*, *per task*, or similar).
 
-When you [branch](../other/glossary.md#branch) an environment to create a new child environment,
-its parent is the environment it was created from.
-If you push a branch through Git or a [source integration](../integrations/source/_index.md),
-the parent defaults to the default environment.
+-   **Testing**: An operational test environment, a user test environment, and a few unit test environments.
 
-You can always [change an environment's parent](./change-parent.md).
+-   **Hotfix**: One environment for each bug, security issue, or hotfix that needs deployment.
 
-## Workflows
+#### Example workflow
 
-Since you can organize your environments as you want, you can create your own workflows.
-There are no rules you must follow when branching environments.
+Example Agile workflow:
 
-You can choose a structure that might best fit your workflow.
-Some possible approaches:
+1.  An admin [branches](../other/glossary.md#branch) the Live (default) environment to create a Sprint environment.
 
-- **Agile**: A child environment per sprint.
-  Each story in the sprint can have its own environment as a child of the sprint environment.
-- **Developer-centric**: One QA environment and a few development environments
-  (*per developer*, *per task*, or similar).
-- **Testing**: An operational test environment, a user test environment, and a few unit test environments.
-- **Hotfix**: One environment for every bug, security, or hotfix that needs deployment.
+2.  The admin gives each developer permission to branch the Sprint environment to create new feature environments.
 
-Here is an example of a possible Agile workflow.
+    ![Agile branches with Live as the top parent, Sprint as a child, and Feature 1 and Feature 2 as children of Sprint](/images/workflow/branches.svg "0.2")
 
-The administrator started by [branching](../other/glossary.md#branch) the Live environment to create a Sprint environment.
-They then give each developer permission to branch the Sprint environment to create new feature environments.
+3.  Feature 1 is developed and work is reviewed by accessing the deployed Feature 1 environment.
 
-![Agile branches with Live as the top parent, Sprint as a child, and Feature 1 and Feature 2 as children of Sprint](/images/workflow/branches.svg "0.2")
+4.  When the review is done, Feature 1 is [merged](../other/glossary.md#merge) into the Sprint environment.
```

Comment:
:nail_care: I still feel like an image of the merge would help make it feel consistent.